content/runbooks/kubernetes/KubeHpaReplicasMismatch.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Kube HPA  Replicas Mismatch
+weight: 20
+---
+
+# KubeHpaReplicasMismatch
+
+## Meaning
+
+Horizontal Pod Autoscaler has not matched the desired number of replicas for
+longer than 15 minutes.
+
+HPA was unable to schedule desired number of pods.
```

Comment:
That's the direct impact, so let's use it instead of "Possible service degradation"